vrc-get-gui/locales/ja.json5

```
@@ -257,6 +257,8 @@
     "projects:manage:button:apply changes": "変更を適用",
     "projects:manage:button:apply": "適用",
 
+    "projects:manage:toast:package_already_installed_as_unlocked": "パッケージ{{package}}はすでにインストールされていますが、VPMによって管理されていません。ALCOMでインストールするには先に手動で削除してください",
```

EN同様+
```suggestion
    "projects:manage:toast:package_already_installed_as_unlocked": "{{package}}は既にインストールされていますが、VPMで管理されていません。ALCOMでインストールするには、先に手動でパッケージを削除してください。",
```
インストールボタンを押した時に出るトーストと思って書いてますが、状況が違ったら教えてください。

あと、もしプロジェクトのPackagesに入っていることが検知出来るのであれば、表示の仕方を変えれば削除or上書きさせることは出来そうです? (バージョンのところにローカルとか入れないといけないので割と面倒そう。)
この場合、アンインストールダイアログに、ローカルだけど本当に消して/上書きして大丈夫? の警告を出す必要がありそう。
アイデアのみ

---

> インストールボタンを押した時に出るトーストと思って書いてますが、状況が違ったら教えてください。

あってます

> あと、もしプロジェクトのPackagesに入っていることが検知出来るのであれば、表示の仕方を変えれば削除or上書きさせることは出来そうです? (バージョンのところにローカルとか入れないといけないので割と面倒そう。)

技術的には可能ではあるけど仕組み的には現状簡単でなはいので一旦エラーで。issue書いてください

---

c774b8e9ce0a2fd1df8ad866c3e3953d6142d0e0